docs/configuration/http_filters/rate_limit_filter.rst

```diff
@@ -33,6 +34,11 @@ stage
 
   **NOTE:** The filter supports a range of 0 - 10 inclusively for stage numbers.
 
+request_type
+  *(optional, string)* The type of requests the filter should apply to. The supported
+  types are *internal*, *external* or *both*. The filter defaults to *both*, and it
```

Comment:
It's not obviously clear what internal and external mean. Describe more and/or link to other docs.